src/common/components/Toast/utils/setToast.ts

```diff
@@ -0,0 +1,17 @@
+import { Bounce, toast } from "react-toastify";
+
+type TostType = "info" | "success" | "warn" | "error";
+
+export default function setToast(type: TostType, text: string): void {
```

Comment:
리턴타입도 의논해서 잡을거면 모두 잡고, 아니면 자동추론으로 하면 좋을 것 같습니다~
```suggestion
export default function setToast(type: TostType, text: string): void {
```